db/models/Location.js

```diff
@@ -2,20 +2,32 @@ import mongoose from "mongoose";
 
 const { Schema } = mongoose;
 
+const infoChildSchema = new Schema({
+  sport: { type: String, required: true },
+  numberOfCourts: { type: Number, required: false },
+  surface: { type: String, required: false },
+});
+
 const locationSchema = new Schema({
   id: { type: Number, required: true },
   title: { type: String, required: true },
-  slug: { type: String, required: true },
-  tags: { type: [String], required: false },
-  rating: { type: Number, required: false },
+  info: [infoChildSchema],
```

Comment:
ich nehme an, diese Schreibweise sagt, du hast 1 oder mehr Objekte des Typs infoChildSchema in einem Array?

Reply:
genau richtig, das habe ich mir von Philip abgeguckt 🤫